x-pack/plugins/index_management/public/services/navigation.js

```diff
@@ -11,6 +11,9 @@ export const setUrlService = (aUrlService) => {
 export const getUrlService = () => {
   return urlService;
 };
-export const getFilteredIndicesUri = (filter) => {
-  return encodeURI(`#${BASE_PATH}indices/filter/${encodeURIComponent(filter)}`);
-};
\ No newline at end of file
+export const getIndexListUri = (filter) => {
+  if(filter) {
+    return encodeURI(`#${BASE_PATH}indices/filter/${encodeURIComponent(filter)}`);
```

Comment:
Can we add a comment to explain the purpose of double encoding?

```js
// React router tries to decode url params but it can’t because the browser partially 
// decodes them. So we have to encode both the URL and the filter to get it all to 
// work correctly for filters with URL unsafe characters in them.
```